### PR TITLE
Added useAppbaseApi option

### PR DESCRIPTION
**Pull request by mahdiyari**
by adding useAppbaseApi : true as an option to the client, dsteem will work with appbase

### CODE REVIEW

**Comment by mahdiyari** [comment]
Tested and edited first commit!

File: src/client.ts [hunk 1]
```diff
@@ -230,11 +242,13 @@ export class Client {
      *
      */
     public async call(api: string, method: string, params: any[] = []): Promise<any> {
+        let callApi = api
+        if (this.useAppbaseApi) callApi = 'condenser_api'
```
Comment (jnordberg) [inline src/client.ts]: This naive approach will break programs that use APIs other than `condenser_api`

Comment (jnordberg) [reply src/client.ts]: This is a much better approach https://github.com/steemit/dsteem/pull/3

Will merge that instead

Comment (mahdiyari) [reply src/client.ts]: I think all appbase APIs are available with condenser_api
Anyway, I think steemit#3 will work too

Comment (therealwolf42) [reply src/client.ts]: I've recently rewritten my code with client.call('condenser_api', ....).

With 0.10.0, this seems to be obsolete, right?

Comment (jnordberg) [reply src/client.ts]: That's right. I'm closing this. Thanks for the PR nevertheless @mahdiyari

**Pull request closed without merging**